lib/fog/vsphere/requests/compute/vm_clone.rb

```diff
@@ -789,14 +789,17 @@ def modify_template_nics_specs(vm_mob_ref, nics, datacenter)
           specs
         end
 
-        def modify_template_volumes_specs(vm_mob_ref, volumes)
+        def modify_template_volumes_specs(vm_mob_ref, volumes, default_storage_pod = nil)
```

Comment:
can we make this a kwarg so it's easier to figure out what the method expects?

Reply:
yeah, feels better :+1: